src/main/java/net/william278/velocitab/tab/PlayerTabList.java

```diff
@@ -244,4 +246,11 @@ public Optional<List<String>> getSiblings(String serverName) {
                     return Optional.of(this.fallbackServers.stream().toList());
                 });
     }
+
+    @Subscribe
+    public void proxyReload(@NotNull ProxyReloadEvent event) {
+        plugin.loadSettings();
+        reloadUpdate();
+        plugin.getServer().getConsoleCommandSource().sendMessage(Component.text("Velocitab has been reloaded!").color(TextColor.color(255, 199, 31)));
```

Comment:
I think i'd rather just use `plugin#log(Level.INFO, "blah")` here tbh :)

Reply:
Also, using a custom `TextColor` will not work since Velocity currently only supports the colors of the `NamedTextColor` class to display in the console

Reply:
I will change it when I get back to my computer.